cypress/integration/login.spec.js

```diff
@@ -0,0 +1,88 @@
+
+import Navbar from '../page-objects/components/Navbar'
+import LoginPage from '../page-objects/pages/LoginPage'
+
+
+describe('Login Success Test', () => {
+    before(function () {
+        cy.visit('http://localhost/kino/')
+        Navbar.clickOnLogin()
+        cy.fixture('login.json').as('login');
+    })
+
+    it('Login with previously manually registered user credentials', function () {
+        cy.login()
+            
+    });
+})
+
+describe('Logout Test', () => {
+        before(function () {
+            cy.visit('http://localhost/kino/')
+            Navbar.clickOnLogin()
+            cy.fixture('login.json').as('login');
+        })
+
+    it('Logout properly', function () {
+        cy.login()
+
+        
+        cy.get('.nav-link').contains('ODJAVA')
+            .click({ force: true });

```

Comment:
Sometimes you are using ; for statement end, other times you don't use them. Consistency!